app.py

```diff
@@ -1,7 +1,7 @@
 from flask import Flask, request, jsonify, render_template, Response
 import redis
 # from sklearn.externals import joblib as pickle
-import cPickle as pickle  # 
+import _pickle as cPickle
```

Comment:
Need to reference as `pickle`. A better way would probably be to do something like this:

```
try: import cPickle as pickle
except: import pickle
```
which would try to import `cPickle` in the case of Python 2 and if that fails it'd fall back to `pickle` in Python 3 which is what we want.